tools/StaticAnalysis/Exceptions/AzureRM.Compute/BreakingChangeIssues.csv

```diff
@@ -99,4 +99,6 @@
 "Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Extension.Diagnostics.AddAzureRmVmssDiagnosticsExtension","Add-AzureRmVmssDiagnosticsExtension","0","1020","The cmdlet 'Add-AzureRmVmssDiagnosticsExtension' no longer has output type 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSet'.","Make cmdlet 'Add-AzureRmVmssDiagnosticsExtension' return type 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSet'."
 "Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Extension.Diagnostics.AddAzureRmVmssDiagnosticsExtension","Add-AzureRmVmssDiagnosticsExtension","0","2020","The cmdlet 'Add-AzureRmVmssDiagnosticsExtension' no longer supports the type 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSet' for parameter 'VirtualMachineScaleSet'.","Change the type for parameter 'VirtualMachineScaleSet' back to 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSet'."
-"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Extension.AzureDiskEncryption.RemoveAzureVmssDiskEncryptionCommand","Disable-AzureRmVmssDiskEncryption","0","3000","The type of property 'Identity' of type 'Microsoft.Azure.Commands.Compute.Automation.Models.PSVirtualMachineScaleSet' has changed from 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSetIdentity' to 'Microsoft.Azure.Commands.Compute.Automation.Models.PSVirtualMachineScaleSetIdentity'.","Change the type of property 'Identity' back to 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSetIdentity'."
\ No newline at end of file
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Extension.AzureDiskEncryption.RemoveAzureVmssDiskEncryptionCommand","Disable-AzureRmVmssDiskEncryption","0","3000","The type of property 'Identity' of type 'Microsoft.Azure.Commands.Compute.Automation.Models.PSVirtualMachineScaleSet' has changed from 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSetIdentity' to 'Microsoft.Azure.Commands.Compute.Automation.Models.PSVirtualMachineScaleSetIdentity'.","Change the type of property 'Identity' back to 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSetIdentity'."
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Extension.Diagnostics.AddAzureRmVmssDiagnosticsExtension","Add-AzureRmVmssDiagnosticsExtension","0","3000","The type of property 'UpgradePolicy' of type 'Microsoft.Azure.Commands.Compute.Automation.Models.PSVirtualMachineScaleSet' has changed from 'Microsoft.Azure.Management.Compute.Models.UpgradePolicy' to 'Microsoft.Azure.Commands.Compute.Automation.Models.PSUpgradePolicy'.","Change the type of property 'UpgradePolicy' back to 'Microsoft.Azure.Management.Compute.Models.UpgradePolicy'."
```

Comment:
@hyonholee I believe that the two exceptions flagged here are false-positives (couldn't find any removed properties), but I want to double-check that there are no properties in the mentioned types have been removed when going from `FooComputeType` to `PSFooComputeType`.